pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -286,4 +291,10 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.PodLogsDir == "" {
 		obj.PodLogsDir = DefaultPodLogsDir
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletCrashLoopBackOffMax) {
+		if obj.CrashLoopBackOff.MaxContainerRestartPeriod == nil {
+			obj.CrashLoopBackOff.MaxContainerRestartPeriod = &metav1.Duration{Duration: MaxContainerBackOff}
```

Comment:
why is this not being defaulted and this reaches validation as nil?

Reply:
worked offline with Laura, the problem is that the feature gate does not get the value at this point, so this path is never exercised